avvylib/src/main/java/xyz/schwaab/avvylib/AvatarView.kt

```diff
@@ -481,13 +502,24 @@ class AvatarView : ImageView {
     }
 
     override fun onDraw(canvas: Canvas) {
-        if (avatarDrawable == null) {
+        if (avatarDrawable == null &&
+                initials == null) {
             return
         }
         if (avatarBackgroundColor != Color.TRANSPARENT) {
             canvas.drawCircle(avatarDrawableRect.centerX(), avatarDrawableRect.centerY(), drawableRadius, circleBackgroundPaint)
         }
-        canvas.drawCircle(avatarDrawableRect.centerX(), avatarDrawableRect.centerY(), drawableRadius, bitmapPaint)
+        if(null != initials){
```

Comment:
If, for some reason, there are both avatar drawable and text set. I would prioritise the avatar itself over the initials.

Any use-case or reason for doing the opposite?

Reply:
No particular reason at all, what do you think of having them both, like
`if(null != avatarDrawable) {`
`canvas.drawCircle(avatarDrawableRect.centerX(), avatarDrawableRect.centerY(),` `drawableRadius, bitmapPaint)`
 `}`

`if(null != initials){`
 `canvas.drawText()`
`}`

Reply:
I think the default, expected behaviour, is to display initials when profile pic is not available.

Prioritising Avatar is specially handy when using something like Picasso, or Volley, etc:
```kotlin
avatarView.text = user.name
Picasso.get().load(user.profilePictureUrl).into(avatarView)
```

And it would work fine. If there's any failure during the loading of the image, the user's name would take position.

Instead of having to attempt to load the image, see the outcome, set text in case it fails.

Also, if `user.profilePictureUrl` is `null`, Picasso (for example) would just ignore it.

This also avoids users of this lib to have to do some checks and clearing of the `text` during recycling:
```kotlin
user.profilePictureUrl?.let{ picUrl ->
    Picasso.get().load(picUrl).into(avatarView)
    avatarView.text = null
} ?: run {
    avatarView.text = user.name
}
```

Reply:
Updated the PR